ComfySoda_Unity2018.2.7f1/Assets/Scripts/ComySodaEngine/EnemyController.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Unity;
+using UnityEditor;
+using UnityEngine;
+
+
+namespace cs
+{
+    public class EnemyController : MonoBehaviour
```

Comment:
"Controller" is typically a per instance thing whereas "Manager" or "System" are usually singletons.